src/haz3lcore/statics/MakeTerm.re

```diff
@@ -286,6 +286,7 @@ and exp_term: unsorted => (UExp.term, list(Id.t)) = {
           | (["-."], []) => BinOp(Float(Minus), l, r)
           | (["*."], []) => BinOp(Float(Times), l, r)
           | (["/."], []) => BinOp(Float(Divide), l, r)
+          | (["**."], []) => BinOp(Float(Power), l, r)
```

Comment:
`**` seems to be only partially implemented

Reply:
Added integer power operator.